deployment/deploy-data

```diff
@@ -6,72 +6,90 @@ set -o errexit
     echo "ERROR: Please only enter 'local', 'beta' or 'production' as environment."
     exit 1
 fi
 
-# Expects path to release directory as second argument. Removes trailing / if present.
-RELEASEDIRECTORYPATH=$2
-if [ "${RELEASEDIRECTORYPATH: -1}" == "/" ] ;then
-    RELEASEDIRECTORYPATH=${RELEASEDIRECTORYPATH%?}
-fi
-
-RELEASEDIRECTORY=$(basename $RELEASEDIRECTORYPATH)
+# Expects path to release archive as second argument. Removes trailing / if present.
+RELEASE_ARCHIVE_PATH=$2
 
 if [ "${ENVIRONMENT}" == "local" ] ;then
-	# Get deployment directory to have relative path to addrelease script
-	DEPLOYMENTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
-	# Get absolute release file paths
-	cd "$RELEASEDIRECTORYPATH"
-	ABSOLUTERELEASEDIR="$PWD"
-    DATA=$ABSOLUTERELEASEDIR/built_with_change_types.tsv
-	REMOVED=$ABSOLUTERELEASEDIR/removed.tsv
-	METADATA=$ABSOLUTERELEASEDIR/version.json
-	DIFFJSON=$ABSOLUTERELEASEDIR/diff.json
-	REPORTS=$ABSOLUTERELEASEDIR/reports.tsv
-
-	# Call addrelease
-	cd "$DEPLOYMENTDIR"
-	python ../website/django/manage.py addrelease ${DATA} ${REPORTS} ${METADATA} ${REMOVED} ${DIFFJSON}
+    # Get deployment directory to have relative path to addrelease script
+    DEPLOYMENTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+    cd "$DEPLOYMENTDIR"
+
+    add_release_tar_to_db /tmp/brca_tar_staging ${RELEASE_ARCHIVE_PATH} "../website/django/manage.py"
 fi
 
 if [ "${ENVIRONMENT}" == "beta" ] || [ "${ENVIRONMENT}" == "production" ] ;then
 
-	REMOTERELEASES=/var/www/backend/${ENVIRONMENT}/django/downloads/releases
-
-	# Copies directory to downloads/releases/
-	rsync -a --rsync-path='sudo rsync' $RELEASEDIRECTORYPATH ${USER}@${HOST}:$REMOTERELEASES/
-
-	# Locations of files on remote server
-	DATA=$REMOTERELEASES/$RELEASEDIRECTORY/built_with_change_types.tsv
-	REMOVED=$REMOTERELEASES/$RELEASEDIRECTORY/removed.tsv
-	METADATA=$REMOTERELEASES/$RELEASEDIRECTORY/version.json
-	DIFFJSON=$REMOTERELEASES/$RELEASEDIRECTORY/diff.json
-	REPORTS=$REMOTERELEASES/$RELEASEDIRECTORY/reports.tsv
-
-	ssh -l${USER} ${HOST} <<-ENDSSH
-	    set -o errexit
-	    . /var/www/backend/${ENVIRONMENT}/virtualenv/bin/activate
-	    python /var/www/backend/${ENVIRONMENT}/django/manage.py addrelease ${DATA} ${REPORTS} ${METADATA} ${REMOVED} ${DIFFJSON}
-	    sudo rm /var/www/backend/${ENVIRONMENT}/django/downloads/releases/current_release.tar.gz
-	    find /var/www/backend/${ENVIRONMENT}/django/downloads/releases/${RELEASEDIRECTORY}/ -name 'release*.tar.gz' -exec cp {} /var/www/backend/${ENVIRONMENT}/django/downloads/releases/current_release.tar.gz  \;
-	ENDSSH
+    REMOTERELEASES=/var/www/backend/${ENVIRONMENT}/django/downloads/releases
+
+    # directory to unpack release archive on the deployment host
+    REMOTE_STAGING=/home/brca/deploy_staging
+
+    RELEASE_DIRECTORY_NAME=$(basename ${RELEASE_ARCHIVE_PATH} .tar.gz)
+
+    # Copies tar to downloads/releases/
```

Comment:
this comment should be changed since this rsync doesn't actually move anything to `downloads/releases/`